public/test/matchers/toEmitValues.test.ts

```diff
@@ -0,0 +1,139 @@
+import { interval, Observable, of, throwError } from 'rxjs';
+import { map, mergeMap, take } from 'rxjs/operators';
+import { OBSERVABLE_TEST_TIMEOUT_IN_MS } from './types';
+
+describe('toEmitValues matcher', () => {
```

Comment:
![mindblown](https://media.giphy.com/media/p3ZV46zdstXxu/giphy.gif)

Reply:
Did I do something unclear here, could I simplify it? Can you elaborate on the mind blown thing @mckn?

Reply:
Love that you are using the extension to test the extension <3

Reply:
❤️